x-pack/plugin/src/test/resources/rest-api-spec/test/snapshot/10_basic.yml

```diff
@@ -22,6 +22,17 @@ setup:
       cluster.health:
         wait_for_status: green
 
+teardown:
```

Comment:
The reason for the test failure seems to be the missing `---` line before teardown. the parser skipped the teardown line I think and was considering snapshot.delete an executable command like do etc which obviously snapshot.delete isn't.